river/LockManager.zig

```diff
@@ -207,7 +209,9 @@ fn handleUnlock(listener: *wl.Listener(void)) void {
     }
 
     if (build_options.xwayland) {
-        server.root.layers.xwayland_override_redirect.node.setEnabled(true);
+        if (server.runtime_xwayland) {
```

Comment:
There's no need to split this in to 2 separate if statements.
```
if (build_options.xwayland and server.xwayland != null) {
```

Reply:
Right, I wasn't sure if two statements would be the same as `if (build_options.xwayland and server.runtime_xwayland)`at buildtime, i.e. would the code be in the build with the second option?

Reply:
They are semantically equivalent in zig, `and` does not evaluate the second operand if the first is false.